docs/release-notes/cloud/cloud-2021-12.md

```diff
@@ -0,0 +1,79 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud December 2021.
+included_jira_versions: ['2021.Q4.6', '2021.Q4.7']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/4.0.22
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/4.0.54
+---
+
+# Cloud December 2021
+
+These release notes are for the Codacy Cloud updates during December 2021.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   It's now possible to [assign a coding standard automatically to new repositories](https://docs.codacy.com/organizations/using-a-coding-standard/#set-default). (CY-5263)
+
+## Bug fixes
+
+-   Fixed an issue that could cause payment plans to change to "Open source" when <span class="skip-vale">Chargebee</span> is unreachable. (CY-5386)
+-   Fixed an issue that sometimes caused repository quality settings to be saved only <span class="skip-vale">partially</span>. (CY-5380)
+-   Fixed an issue that prevented merge commits from displaying any information regarding files, duplication, and complexity. (CY-5364)
+-   Fixed a scenario where the number of applied repositories on a coding standard didn't update when deleting a repository. (CY-5363)
+-   Fixed an issue that prevented non-logged users to see the file list of open source repositories. (CY-5298)
+-   Fixed a popup that overlapped the list on the People page. (CY-5282)
+-   Improved the visual feedback for the Jira integration status. (CY-5190)
+-   Fixed an issue that prevented the message "Refresh the page to see the results" from being displayed on the commit and pull request pages after a re-analysis was completed. (CY-5187)
```

Comment:
This one is already in November notes

Reply:
Very good catch @inesgomas! I tend to always forget to check if the release notes for older issues have already been published in the previous month. I should find a way to detect these situations automatically. :nerd_face: